pycon.py

```diff
@@ -140,6 +166,10 @@ def _get_template_variables(**kwargs):
     }
     variables.update(kwargs)
 
+    cover_name = kwargs.get('cover_name')
+    if cover_name:
+        variables.update(**get_cover(name=cover_name))
```

Comment:
Do we need this? Why don`t we do all the logic in `get_cover` method?

I like that you have introduced `get_cover` method you can pass `kwargs.get('cover_name', None)` into it. Inside you can check if cover_name is None use default otherwise load defined image... This way we get rid of one if statement in the code...

Reply:
If we do my last comment (with css this is irrelevant)